PSAP-1003: NTO: reapply_sysctl and verify Profiles improvements (#56773)
openshift/openshift-docs: modules/cluster-node-tuning-operator-verify-profiles.adoc

* `APPLIED`: `True` if the TuneD daemon applied the desired profile. (`True/False/Unknown`).
* `DEGRADED`: `True` if any errors were reported during application of the TuneD profile (`True/False/Unknown`).
* `AGE`: Time elapsed since the creation of Profile object.

> The ClusterOperator/node-tuning object also contains useful information about the Operator and its node agents' health. For example, Operator misconfiguration will be reported via ClusterOperator/node-tuning status messages.

Reviewer: ```suggestion
The `ClusterOperator/node-tuning` object also contains useful information about the Operator and its node agents' health. For example, Operator misconfiguration is reported by `ClusterOperator/node-tuning` status messages.
```

> 

Reviewer: Needs an introductory sentence before the code block, such as "To get status information about the `ClusterOperator/node-tuning` object, run the following command:"

[source,terminal]
----
$ oc get co/node-tuning -n openshift-cluster-node-tuning-operator
----

.Example output
[source,terminal]
----
NAME          VERSION   AVAILABLE   PROGRESSING   DEGRADED   SINCE   MESSAGE
node-tuning   4.13.1    True        False         True       60m     1/5 Profiles with bootcmdline conflict
----

> If either the ClusterOperator/node-tuning or a Profile object's status is `DEGRADED`, additional information will be provided in the operator or operand logs.

Reviewer: ```suggestion
If either the `ClusterOperator/node-tuning` or a profile object's status is `DEGRADED`, additional information is provided in the Operator or operand logs.
```

* `APPLIED`: `True` if the TuneD daemon applied the desired profile. (`True/False/Unknown`).
* `DEGRADED`: `True` if any errors were reported during application of the TuneD profile (`True/False/Unknown`).
* `AGE`: Time elapsed since the creation of Profile object.

The ClusterOperator/node-tuning object also contains useful information about the Operator and its node agents' health.

> For example, operator misconfiguration will be reported via ClusterOperator/node-tuning status messages.

Reviewer 2: There's a couple places where you've written "operator" and "Operator". Guidelines require that it be written capitalized Operator throughout.

Author: Ack.  And removed a hard-wrap here too.

[source,terminal]
----
$ oc get co/node-tuning -n openshift-cluster-node-tuning-operator
----

.Example output
[source,terminal]
----
NAME          VERSION   AVAILABLE   PROGRESSING   DEGRADED   SINCE   MESSAGE
node-tuning   4.13.1    True        False         True       60m     1/5 Profiles with bootcmdline conflict
----

> If either the ClusterOperator/node-tuning or a Profile object's status

Reviewer 2: Can you remove the hard wrap? Our [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-file-guidelines) recommend undoing it where possible.

> is Degraded, additional information will be provided in the operator or

Reviewer 2: Let's change this to `DEGRADED` in backticks to match line 29 above.
